src/routes/paths.ts

```diff
@@ -3,6 +3,7 @@ import type { Location } from 'react-router-dom';
 import { APP_PATH_CREATE_CIRCLE } from 'utils/domain';
 
 export const EXTERNAL_URL_DOCS = 'https://docs.coordinape.com';
+export const EXTERNAL_URL_TOS = 'https://coordinape.com/terms-of-service';
```

Comment:
export const EXTERNAL_URL_TOS = 'https://coordinape.com/terms-of-service';
```suggestion
export const EXTERNAL_URL_TOS = 'https://coordinape.com/terms';
```